Stripe/PublicHeaders/STPSetupIntent.h

```diff
@@ -0,0 +1,89 @@
+//
+//  STPSetupIntent.h
+//  StripeiOS
+//
+//  Created by Yuki Tokuhiro on 6/27/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+#import "STPSetupIntentEnums.h"
+
+@class STPIntentAction;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A SetupIntent guides you through the process of setting up a customer's payment credentials for future payments.
+
+ @see https://stripe.com/docs/api/setup_intents
+ */
+@interface STPSetupIntent : NSObject<STPAPIResponseDecodable>
+
+/**
+ The Stripe ID of the SetupIntent.
+ */
+@property (nonatomic, readonly) NSString *stripeId;
```

Comment:
I think we should start standardizing on `stripeID` instead of `stripeId`. Any thoughts there?

Reply:
Fine with either, if only we could rename everything to be consistent..